.vscode/launch.json

```diff
@@ -200,7 +200,8 @@
 			"outFiles": [
 				"${workspaceRoot}/src/**/*.js"
             ],
-            "sourceMaps": true
+            "sourceMaps": true,
+            "preLaunchTask": "build-smoke-tests"
```

Comment:
Can we add `--dont-delete-vsix` argument to the `launch.json` Launch Smoke Test debug configuration? It for the local tests, so I think it should be there